triage/model_trainers.py

```diff
@@ -14,6 +15,29 @@
     db_retry
 
 
+def _ad_hoc_feature_importances(model):
+    """
+    Get the "ad-hoc feature importances" for
+
+    Args:
+        model: A trained model that has not a `feature_importances_` attribute
+
+    Returns:
+        At this moment, this method only returns the odds ratio of both the
+        intercept and the coefficients given by sklearn's implementation of
+        the LogisticRegression.
+        The order of the odds ratio list is the standard
+        of the statistical packages (like R, SAS, etc) i.e. (intercept, coefficients)
+    """
+    feature_importances = None
+
+    if isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)):
+        coef_odds_ratio = np.exp(model.coef_)
+        intercept_odds_ratio = np.exp(model.intercept_)
+        feature_importances = np.hstack((intercept_odds_ratio, coef_odds_ratio))
+
+    return feature_importances
+
 def get_feature_importances(model):
```

Comment:
Now that there is some real logic here, I think a unit test would help it. Particularly, a test case for LogisticRegression with some dummy values for `coef_` and `intercept_` and make sure this code path gets called.